client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -73,10 +73,11 @@ function getHoldMessages( siteSlug, translate ) {
 			),
 		},
 		EXCESSIVE_DISK_SPACE: {
-			title: translate( 'Site too large' ),
+			title: translate( 'Site not supported' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Private site not supported' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/atomic-disk-space-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).